monitoring/voltage-monitoring/src/main/java/com/farao_community/farao/monitoring/voltage_monitoring/VoltageMonitoringResult.java

```diff
@@ -21,9 +24,12 @@ public class VoltageMonitoringResult {
 
     public enum Status {
         SECURE,
+        UNSECURE,
         HIGH_VOLTAGE_CONSTRAINT,
         LOW_VOLTAGE_CONSTRAINT,
-        HIGH_AND_LOW_VOLTAGE_CONSTRAINTS;
+        HIGH_AND_LOW_VOLTAGE_CONSTRAINTS,
+        DIVERGENT,
```

Comment:
This is not clear in the US as the Status DIVERGENT exist in the diagram. But the reference is the text above it. 
And in the text, this status does not exist : If the initial loadflow diverges, we expect an UNKNOWN status. Therefore, you can remove it.